ibm/service/eventnotification/resource_ibm_en_destination_custom_email.go

```diff
@@ -52,6 +52,11 @@ func ResourceIBMEnCustomEmailDestination() *schema.Resource {
 				Optional:    true,
 				Description: "Whether to collect the failed event in Cloud Object Storage bucket",
 			},
+			"verification_type": &schema.Schema{
+				Type:        schema.TypeString,
+				Optional:    true,
+				Description: "Verification Method spf/dkim.",
```

Comment:
Can we add validation on supported methods

Reply:
Added validator for the same in latest commit